include/glow/Graph/Context.h

```diff
@@ -37,19 +38,43 @@ class Context final {
 public:
   /// Maps placeholders to the tensors that back them.
   using PlaceholderMap = std::unordered_map<Placeholder *, Tensor *>;
+  /// Additional Tensors used for this graph, and whether the memory is owned by
```

Comment:
can you clarify what those "additional" tensors are.

Reply:
I was trying to keep it generic at the Context level, as it could be different for different backends. For the Interpreter, the running of a network appears to require Tensors that aren't linked to existing Placeholders. Haven't looked into why exactly, but guessing it's intermediate tensors.